test/common/config.spec.ts

```diff
@@ -73,5 +75,15 @@ describe('extractNamespaceName()', () => {
     expect(config.SKIP_K8S_JOBS).toEqual(false);
     expect(config.WORKERS_COUNT).toEqual(10);
     expect(config.SKOPEO_COMPRESSION_LEVEL).toEqual(6);
+    expect(config.SYSDIG_ENDPOINT).toEqual('https://api/v1/images/');
+    expect(config.SYSDIG_TOKEN).toEqual('1432gtrhtrw32raf');
+    delete process.env.SNYK_SYSDIG_ENDPOINT;
+    delete process.env.SNYK_SYSDIG_TOKEN;
```

Comment:
Nit: the `delete`s should be in the test below, otherwise there is a dependency between the two tests